patches/remove-dlink.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/targets/ar71xx-tiny b/targets/ar71xx-tiny
```

Comment:
Wie kommt denn dieses diff zustande? :D

Reply:
Wie meinst du das?

Reply:
Hab mich nur gewundert weil der dlink entfernt wird :)

Reply:
Bei uns in München passt die Fw nicht mehr drauf. Daher hatte ich den hier auch gestrichen, aber wenn es passt, können wir das auch so lassen

Reply:
ich glaub den nutzt eh keiner. also lassen wir den mal drinnen :D